common/lib/xmodule/xmodule/partitions/tests/test_partitions.py

```diff
@@ -110,6 +110,7 @@ class PartitionTestCase(TestCase):
     TEST_ID = 0
     TEST_NAME = "Mock Partition"
     TEST_DESCRIPTION = "for testing purposes"
+    TEST_PARAMETERS = {"location": "block-v1:edX+DemoX+Demo+type@block@uuid"}
```

Comment:
"Location" is the term we've been using internally for verification checkpoints, so I'd prefer to call it that for consistency.